magento/magento2
# Pass website id in the post to resolve this issue.

### Description (*)


Pass website id in the post to resolve this issue.

### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#27098: Magento CE 2.3.X the swagger interface api named integrationCustomerTokenServiceV1 http scheme valid customer acess details entered and execute still server response is 401

### Manual testing scenarios (*)


1. Open the default Magento Swagger  HTTP schemes or use this same API in postman.
2. Open integrationCustomerTokenServiceV1 interface to generate a token for customers.
3. Enter valid customers details and check the response.

### Questions or comments


### Contribution checklist (*)
